cmd/getDefaults.go

```diff
@@ -81,9 +82,6 @@ func getDefaults() ([]map[string]string, error) {
 			var yamlContent string
 
 			if !defaultsOptions.useV1 {
-				log.Entry().Warning("This step is using deprecated format of stage conditions which will be removed in Jan 2024. " +
-					"To avoid pipeline breakage, please call getDefaults command with --useV1 flag.",
-				)
```

Comment:
`getDefaults` step is used by piper-azure-task for downloading piper-defaults (old format) and stage-config (new format). No need to keep the warning message here since we are not going to move piper-defaults to new format